package.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "custom",
+  "version": "1.0.0",
+  "scripts": {
+    "dev": "vite",
+    "build": "tsc && vite build"
+  },
+  "dependencies": {},
+  "devDependencies": {
+    "lit": "2.0.0-rc.3",
```

Comment:
We could use a relased version, even if I know there is no consequence to keep it. :-)

Reply:
+1 :-)